java-security-test/src/test/java/com/sap/cloud/security/test/JwtGeneratorTest.java

```diff
@@ -335,7 +335,7 @@ public void getInstanceFromFile_loadsJsonData() throws IOException {
 		assertThat(token.getClaimAsString(TokenClaims.XSUAA.ZONE_ID)).isEqualTo("zone-id");
 		assertThat(token.getClaimAsStringList(TokenClaims.XSUAA.SCOPES)).containsExactlyInAnyOrder("openid",
 				"app1.scope");
-		assertThat(token.getClaimAsString(TokenClaims.XSUAA.CLIENT_ID)).isEqualTo("testingClientId");
+		assertThat(token.getClientId()).isEqualTo("testingClientId");
```

Comment:
actually you can test both ?
add this test case as well?

`assertThat(token.getClaimAsString(TokenClaims.XSUAA.CLIENT_ID)).isEqualTo("testingC`lientId");